Plutarch/Api/V1/AssocMap.hs

```diff
@@ -56,3 +61,88 @@ instance
       x' <- Plutus.fromData x
       y' <- Plutus.fromData y
       Just (x', y')
+
+----------------------- Smart constructor to to create onchain instances ----------------
+
+{- |
+    Smart constructor to enforce PMap invariants on the Haskell level for writing
+    onchain code
+-}
+pmkPMap ::
```

Comment:
This function seems weird to me. You could just take an actual map from `containers` as input

Reply:
copied.